lib/car.rb

```diff
@@ -0,0 +1,28 @@
+class Car
+  attr_reader :monthly_payment,
+              :loan_length,
+              :color,
+              :make,
+              :model
+
+
+
+  def initialize(name, monthly_payment, loan_length)
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    @name = ("#{@make}#{@model}")
+    @make = "Ford"     # I couldnt figure out how to make this dynamic so I
+    @model = "Mustang" # hard coded it so i could move forward
```

Comment:
There are a couple options you could have used to make it dynamic:

@make = name.split[0]
@model = name.split[1]

@make = name.split.first
@model = name.split.last